util/comdb2file.c

```diff
@@ -124,6 +124,30 @@ char *comdb2_location(char *type, char *fmt, ...)
     return out;
 }
 
+/* Return location only if it is found in the hash
+ * caller needs to free returned string */
+char *comdb2_location_in_hash(char *type, char *fmt, ...)
+{
+    struct location *l = hash_find_readonly(locations, &type);
+    if (l == NULL) {
+        return NULL;
+    }
+
```

Comment:
I think we don't need the fmt related processing in this function; just the hash lookup should do.

Reply:
Ftr, keeping the behavior like this aligns with the other function. It is currently unnecessary, but possibly useful in future, so no harm in keeping it this way. Thanks.